components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -258,7 +251,7 @@ - (void)commonMDCFlexibleHeaderViewInit {
   _statusBarHintCanOverlapHeader = YES;
 
   _minMaxHeightIncludesSafeArea = YES;
-  _minimumHeight = kFlexibleHeaderDefaultHeight + [self fhv_topSafeAreaInset];
+  _minimumHeight = kFlexibleHeaderDefaultHeight + [_topSafeArea topSafeAreaInset];
```

Comment:
Please consider capturing `[_topSafeArea topSafeAreaInset]` in a local variable unless it's expected to change between the two calls.